src/queries/text.ts

```diff
@@ -37,7 +40,9 @@ const getMultipleError = (text: TextMatch) =>
   `Found multiple elements with text: ${String(text)}`;
 
 const getMissingError = (text: TextMatch) =>
-  `Unable to find an element with text: ${String(text)}`;
+  `Unable to find an element with text: ${String(text)}
+
+${format(screen.toJSON() || [], { mapProps: mapVisibilityRelatedProps })}`;
```

Comment:
Pls add formatting rather in `makeQueries` function, in order to have it affect all query types.

Reply:
I've now moved this formatting to `makeQueries`.